src/components/Table/TableRow.tsx

```diff
@@ -31,14 +33,42 @@ const rowVariants = {
   shown: { opacity: 1 }
 }
 
-const TableRow: FC<RowProps> = ({ children, onClick, className }) => (
-  <motion.tr variants={rowVariants} transition={{ duration: 0.8 }} onClick={onClick} className={className}>
-    {children}
+const TableRow: FC<RowProps> = ({ children, onClick, linkTo, className }) => (
+  <motion.tr variants={rowVariants} transition={{ duration: 0.8 }} onMouseUp={onClick} className={className}>
+    {Children.map(children, (c) =>
```

Comment:
Ha, I wasn't aware of the `Children` API! Looking at the docs I see that this API belongs to legacy and they recommend using alternatives: https://beta.reactjs.org/apis/react/Children

Maybe it's worth having a second look at this implementation. WDYT?

With a quick look, maybe something similar to how `headerTitles` is handled by the `TableHeader`?

Reply:
Crap. I'll take a second look :)

Reply:
TODO added